### PR TITLE
Reimplement WithConsecutiveRector (#311)

### PR DESCRIPTION
**Pull request by f-albert**
#311 I tried to reimplement WithConsecutiveRector because the previous implementation is incorrect.

I think i need to refactor the code.
It's my first time using phpparser and contributing on github. I am open to suggestions for improvements. :)

### CODE REVIEW

**Comment by samsonasik** [comment]
could you run:

```
vendor/bin/rector && composer fix-cs
```

to make CI green? Thank you.

**Comment by f-albert** [comment]
@TomasVotruba When can I expect a review?

File: tests/Rector/StmtsAwareInterface/WithConsecutiveRector/Fixture/exclude_will_return_map.php.inc [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+                [3, 4],
+            )
+            ->willReturnMap([
+                [1, 2, 5],
+                [3, 4, 6],
+            ]);
+    }
+}
+
+?>
```
Comment (samsonasik) [inline tests/Rector/StmtsAwareInterface/WithConsecutiveRector/Fixture/exclude_will_return_map.php.inc:49]:
```suggestion
````

**Review by samsonasik — APPROVED** [review]

**Comment by TomasVotruba** [comment]
Thanks for reviews @samsonasik :+1: 

Let's give this a go :clap:

**Pull request merged**